devicedefender/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.1)
-project(IotDeviceDefender-cpp CXX C)
+
+project(IotDeviceDefender-cpp
+        VERSION ${SIMPLE_VERSION}
```

Comment:
Are these subprojects supposed to support being built standalone? Without being built from the root aws-iot-device-sdk-cpp-v2 root directory? 

If this is built standalone, `${SIMPLE_VERSION}` isn't going to be defined

Reply:
My gut reaction says "no", as they require things like `#include <aws/crt/Api.h>` without setting their CMake file to include `aws-c-common` (for example). However, that said, it looks like *in theory* there is nothing saying it explicitly has to be built using the SDK.

However, it uses things like `aws_use_package` in the CMake file without importing them, in the case of `eventstream_rpc/CMakeLists.txt` - so there is a dependency on being a sub-project to a CMake project that has `aws_use_package` at least.

I think what I will do is try to build a few of the other sub-projects in a standalone build and if they all fail, then I think it is safe to say they need to be built as a CMAKE sub-project to the C++ SDK.

Reply:
Update: It appears it is hit and miss. Device Defender can be built standalone and will have issues due to `SIMPLE_VERSION` not existing, while EventStream RPC is dependent on being a sub-project.

I will look into a way to fix this where we can set the version still, but also support standalone builds. 👍

Reply:
I fixed it! Now each CMake file has a string that is the last known release version that it uses if `SIMPLE_VERSION` is not set. The version string for each sub-project CMake file is automatically updated as part of running `update-versions.sh`

I checked and it works when the project is standalone (falls back to release string) or when it is a direct sub-project (uses `SIMPLE_VERSION`).